core/src/main/scala/com/spotify/featran/FeatureExtractor.scala

```diff
@@ -38,15 +38,16 @@ class FeatureExtractor[M[_]: CollectionType, T] private[featran]
   @transient private val dt: CollectionType[M] = implicitly[CollectionType[M]]
   import dt.Ops._
 
-  @transient private lazy val as: M[(T, ARRAY)] = input.map(o => (o, fs.unsafeGet(o)))
+  private def as(@transient input: M[T], fs: FeatureSet[T]): M[(T, ARRAY)] =
+    input.map(o => (o, fs.unsafeGet(o)))
```

Comment:
Will this be executed twice, once in `aggregate` and once in `featureValuesWithOriginal`? Could be potentially expensive if some feature getter does expensive computation?